pkgs/top-level/all-packages.nix

```diff
@@ -7474,6 +7474,8 @@ in
 
   xfstests = callPackage ../tools/misc/xfstests { };
 
+  xkcdpass = with pythonPackages; toPythonApplication xkcdpass;
```

Comment:
Is this the right section in ``all-packages.nix`` for this?

Reply:
Hmm ... maybe not.

I placed the new package by looking for [the position of `pwgen`](https://github.com/NixOS/nixpkgs/blob/f448ba9f6b404e2314e784a9b2d8a15c30aa3871/pkgs/top-level/all-packages.nix#L6075-L6077) (a tool serving a similar purpose) ...
https://github.com/NixOS/nixpkgs/blob/f448ba9f6b404e2314e784a9b2d8a15c30aa3871/pkgs/top-level/all-packages.nix#L6074-L6078
...  and then scrolling forward to `x...`.

However, if I search for the section titles marked by `###`, it seems that this is in section "DEVELOPMENT / EMSCRIPTEN"
https://github.com/NixOS/nixpkgs/blob/f448ba9f6b404e2314e784a9b2d8a15c30aa3871/pkgs/top-level/all-packages.nix#L3132-L7826
which doesn't seem like the right category.

I guess "TOOLS" would be a better fit?
https://github.com/NixOS/nixpkgs/blob/f448ba9f6b404e2314e784a9b2d8a15c30aa3871/pkgs/top-level/all-packages.nix#L506-L3131